src/ledger/InMemoryLedgerTxn.cpp

```diff
@@ -0,0 +1,59 @@
+// Copyright 2021 Stellar Development Foundation and contributors. Licensed
+// under the Apache License, Version 2.0. See the COPYING file at the root
+// of this distribution or at http://www.apache.org/licenses/LICENSE-2.0
+
+#include "ledger/InMemoryLedgerTxn.h"
+#include "ledger/LedgerTxnImpl.h"
```

Comment:
Why do you actually need this include?

Reply:
For the definition of `stellar::EntryIterator::AbstractImpl`. It's only forward-declared in `LedgerTxn.h`

Reply:
Ah that's unfortunate. Opened https://github.com/stellar/stellar-core/issues/3125 to make this cleaner at some future date.